src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -0,0 +1,174 @@
+
+You can save 100–500 ms by establishing early connections to important third-party origins.
+
+Two [`<link>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) types can help here:
+
+* `preconnect`
+
+* `dns-prefetch`
+
+### `preconnect`
+
+`<link rel="preconnect">` informs the browser that your page intends to establish a connection to another origin, and that you’d like the process to start as soon as possible. When the request for a resource from the preconnected origin is made, the download starts immediately. 
+
+```html
+<link rel="preconnect" href="https://cdn.example.com">
+```
+
+{% Aside 'caution' %}
+
+Only preconnect to critical origins you will use soon because if the connection isn’t used within 10 seconds the browser closes it. There’s a limit to the number of simultaneous connections a browser can handle (for example, Chrome can handle six), and unnecessary preconnecting can delay other important resources.
+
+{% endAside %}
+
+### `dns-prefetch`
+
+`<link rel="dns-prefetch>` handles a small subset of what is handled by `<link rel="preconnect">`.  Establishing a connection involves the DNS lookup and TCP handshake, and for secure origins, TLS negotiations. `dns-prefetch` instructs the browser to only resolve the DNS of a specific domain before it has been explicitly called.
+
+Because there’s a limit to the number of connections a browser can open, for less critical third-party origins use `<link rel=dns-prefetch>`. 
+
+```html
+<link rel="dns-prefetch" href="http://example.com">
+```
+
+[Browser support for `dns-prefetch`](https://caniuse.com/#search=dns-prefetch) is slightly different from [`preconnect` support](https://caniuse.com/#search=preconnect), so `dns-prefetch` can serve as a fallback for browsers that don’t support `preconnect`. Use separate link tags to implement this safely: 
+
+```html
+<link rel="preconnect" href="http://example.com">
+<link rel="dns-prefetch" href="http://example.com">
+```
+## Lazy-load third-party resources
+
+Embedded third-party resources can be a big contributor to slow page speed when constructed poorly. If they aren’t critical or are below the fold (that is, if users have to scroll to view them), lazy-loading is a good way to improve page speed and paint metrics. This way users will get the main page content faster and have a better experience.
+
+<figure class='w-figure w-figure--inline-left'>
+<img src="lazy-load.png" alt="A diagram of a webpage shown on a mobile device with scrollable content extending beyond the screen. The content that's below-the-fold is desaturated because it's not loaded yet.">
+</figure>
+
+One way to do that is to lazy-load third-party content after the main page content loads. Ads are a good candidate for this approach.
```

Comment:
```suggestion
One effective approach is to lazy-load third-party content after the main page content loads. Ads are a good candidate for this approach.
```
To avoid the ambiguous "that."